docs/dev_env/Ubuntu_Development_Environment.md

```diff
@@ -0,0 +1,146 @@
+    ```
+
+### Step 2: Install the Unity Hub:
+
+Follow [these instructions](https://docs.unity3d.com/hub/manual/InstallHub.html#install-hub-linux) for how to install the Unity Hub on Ubuntu. For convenience, these instructions are replicated below:
+
+```bash
+wget -qO - https://hub.unity3d.com/linux/keys/public | gpg --dearmor | sudo tee /usr/share/keyrings/Unity_Technologies_ApS.gpg > /dev/null
+
+sudo sh -c 'echo "deb [signed-by=/usr/share/keyrings/Unity_Technologies_ApS.gpg] https://hub.unity3d.com/linux/repos/deb stable main" > /etc/apt/sources.list.d/unityhub.list'
+
+sudo apt update
+sudo apt-get install unityhub
+```
+
+### Step 3: Configure SSH Access to GitHub (Required):
+
+1. Follow [this](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) guide for creating a new SSH key.
+
+2. Follow [this](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) guide for adding that SSH key to GitHub.
+
+    > [!NOTE]
+    > Test that the connection works by running the following command:
+    >
+    > ```bash
+    > sudo ssh -T git@github.com 
+    > ```
+
+You'll be prompted to trust the key (type yes and hit enter). This command should congratulate you on successfully connecting, list your username, and tell you that command-line access is not enabled.
+
+### Step 4: Clone the EOS Plugin Repository
+
+It is important that you have the latest version of `git` installed, so that you can use `git lfs`. See [here](https://itsfoss.com/install-git-ubuntu/) for an excellent guide an installing the latest version of `git`
+
+> [!NOTE]
+> Simply installing `git` via apt is not sufficient.
+
+```bash
+git clone git@github.com:PlayEveryWare/eos_plugin_for_unity_restricted
+```
+
+It is important that you follow the command format above instead of using something like [`https://github.com`](https://github.com), because it will otherwise fail. The format guarantees that the SSH protocol is used.
+
+Once the repository is cloned, pull the lfs files with the following command from inside the repository:
+
+```bash
+git lfs pull
+```
+
+Create a directory in the root of the project called `Builds`, and inside that directory create two directories: `Server` and `Normal`. These will be utilized later.
+
+### Step 5: Configuring Unity Project for Linux
+
+Once Unity is open:
+
+1. Go to File ➝ Build Settings
+2. Switch the platform to "Dedicated Server."
+3. Click on the button labeled "Player Settings..." at the bottom left.
+4. Go to Tools ➝ EOS Automated Test Settings
+5. Select the previously created `Builds/Server` directory for the "Test Server Directory" field.
+
+## Optional:
+
+Set up a shared folder between Linux and Windows. Follow these instructions: [https://linuxhint.com/shared\_folders\_hypver-v\_ubuntu\_guest/](https://linuxhint.com/shared_folders_hypver-v_ubuntu_guest/)
```

Comment:
Should these be in the Hyper-V section? It seems like you're setting up a clear delineation between Hyper-V setup and regular linux setup with the 2 different files, but there is still a lot of Hyper-V related stuff in the regular linux file. Should these be more separated?

Reply:
Good catch. I moved these to the other doc.